bin/node/cli/Cargo.toml

```diff
@@ -34,13 +34,13 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 # third-party dependencies
+clap = { version = "3.0", features = ["derive"], optional = true }
 codec = { package = "parity-scale-codec", version = "2.0.0" }
 serde = { version = "1.0.132", features = ["derive"] }
 futures = "0.3.16"
 hex-literal = "0.3.4"
 log = "0.4.8"
-rand = "0.7.2"
-structopt = { version = "0.3.25", optional = true }
+rand = "0.8"
```

Comment:
This update isn't related to your changes, right? I would prefer this to be set back to `0.7.2` and left untouched.